pkg/ctl/get/nodegroup.go

```diff
@@ -38,15 +38,18 @@ func getNodeGroupCmd(cmd *cmdutils.Cmd) {
 		cmdutils.AddRegionFlag(fs, &cmd.ProviderConfig)
 		cmdutils.AddCommonFlagsForGetCmd(fs, &params.chunkSize, &params.output)
 		cmdutils.AddTimeoutFlag(fs, &cmd.ProviderConfig.WaitTimeout)
+		cmdutils.AddConfigFileFlag(fs, &cmd.ClusterConfigFile)
 	})
 
 	cmdutils.AddCommonFlagsForAWS(cmd.FlagSetGroup, &cmd.ProviderConfig, false)
 }
 
 func doGetNodeGroup(cmd *cmdutils.Cmd, ng *api.NodeGroup, params *getCmdParams) error {
+	if err := cmdutils.NewMetadataLoader(cmd).Load(); err != nil {
+		return err
+	}
 	cfg := cmd.ClusterConfig
 
-	// TODO: move this into a loader when --config-file gets added to this command
```

Comment:
I think this TODO is saying to move all the validation below into the loader to match the pattern in other `ctl`s 🤔

Reply:
Oh... you might be right. :D

Reply:
Ah I meant that this could be done as part of this PR! WDYT?

Reply:
definitely not :DD

Reply:
why not? Seems related to this PR :thinking: